Source/DafnyLanguageServer.Test/GutterStatus/CachedLinearVerificationGutterStatusTester.cs

```diff
@@ -25,10 +25,10 @@ public class CachedLinearVerificationGutterStatusTester : LinearVerificationGutt
       options.Set(LanguageServer.VerifySnapshots, 1U);
     });
     await VerifyTrace(@"
- .  S  S  |  I  $  | :method test() {
- .  S  |  |  I  $  | :  assert true;
- .  S  S  |  I  $  | :  //Replace: 
- .  S  S  |  I  $  | :}", true);
+ .  S  |  I  $  | :method test() {
+ .  S  |  I  $  | :  assert true;
```

Comment:
Wow, interesting, is it because we don't submit this to the solver anymore?

Reply:
O, actually, I didn't think about this. The new Boogie API we're using does not support caching at all. The reason for that is that caching in Boogie is implemented on an implementation level, and the new Boogie API allows partially verifying implementations, at which point it's not clear when to set the implementation level cache.

It would be not be difficult to update the caching mechanism in Boogie so it works at the VC instead of the implementation level, but since we are not using the caching anyways because we believe it's buggy, I did not do that work. I think it makes more sense to do that when we plan on actually using the caching.